matcher/service/openreview_interface.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, client, config_note_id, logger=logging.getLogger(__name__)):
         self._maximums = None
         self._demands = None
         self._constraints = None
+        self._custom_demand_edges = None
```

Comment:
These two variables shouldn't be properties of the interface. If you see none of the edges are properties of the class. `_get_custom_demand_edges` should be helper functions, you only need to call the helper once. 
The properties `_maximums` , `_minimums` and `_demands` are properties of the class and have the values computed based on edge data.

Reply:
Thanks for explaining it. I agree and I have made changes for this.